docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/authentication-config/configure-azure-ad.md

```diff
@@ -130,38 +131,38 @@ Rancher does not validate the permissions you grant to the app in Azure. We only
 
 1. Obtain your Rancher **Application (Client) ID**.
 
-    1. Use search to open **App registrations** (if not already there).
+    1. If you aren't already there, use search to open **App registrations**.
 
     1. In **Overview**, find the entry you created for Rancher.
 
     1. Copy the **Application (Client) ID** and paste it into Rancher as your **Application ID**.
 
-1. Your endpoint options will typically be either [Standard](#global) or [China](#china). With these options, you only enter the **Tenant ID**, **Application ID**, and **Application Secret**.
+1. In most cases, your endpoint options will either be [Standard](#global) or [China](#china). For either of these options, you only need to enter the **Tenant ID**, **Application ID**, and **Application Secret**.
 
 ![Standard Endpoint Options](/img/tenant-application-id-secret.png)
 
 **For Custom Endpoints:**
 
 **Warning:** Custom Endpoints are not tested or fully supported by Rancher.
```

Comment:
```suggestion
:::caution

Custom Endpoints are not tested or fully supported by Rancher.

:::
```

We should convert this to use Docusaurus' [admonitions](https://docusaurus.io/docs/markdown-features/admonitions) so there's increased visibility.

Reply:
Adjusted!